core/src/i18n/en.yaml

```
@@ -716,12 +716,25 @@ functions:
       text: Warning
       variable-can-override-sbu: This variable can override or be overridden by a variable injected by one of the created ServiceBindings.
 gateways:
+  name_singular: Gateway
+  description: Gateway describes a load balancer operating at the edge of the mesh receiving incoming or outgoing HTTP/TCP connections.
   create-modal:
+    at-least-one-server-required: Gateway must have at least one server.
+    tooltips:
+      ca-certificates: The path to a file containing certificate authority certificates to use in verifying a presented client side certificate.
+      selector: One or more labels that indicate a set of pods on which the gateway configuration should be applied. By default workloads are searched across all namespaces based on label selectors.
+      protocol: The protocol exposed on the port. TLS implies the connection will be routed based on the SNI header to the destination without terminating the TLS connection.
```

```suggestion
      protocol: The protocol exposed on the port. **TLS** implies that the connection is routed based on the SNI header to the destination, without terminating the TLS connection.
```

I assume TLS is an option in a dropdown list/radio box or something similar?

...Can we do bold formatting in yaml?

---

Yup, it'a a list option... and unfortunately, we don't support formatting in translations - this is in the tooltip anyway.